mixer/adapter/stdio/stdio.go

```diff
@@ -139,9 +131,29 @@ func GetInfo() adapter.Info {
 			metric.TemplateName,
 		},
 		DefaultConfig: &config.Params{
-			LogStream:    config.STDOUT,
-			MetricLevel:  config.INFO,
-			OutputAsJson: false,
+			LogStream:                  config.STDOUT,
+			MetricLevel:                config.INFO,
+			OutputLevel:                config.INFO,
+			OutputAsJson:               true,
+			MaxDaysBeforeRotation:      30,
+			MaxMegabytesBeforeRotation: 100 * 1024 * 1024,
```

Comment:
aside: it would be really nice if there was a size equivalent of `time.Seconds`.